edb/server/args.py

```diff
@@ -815,6 +817,26 @@ def resolve_envvar_value(self, ctx: click.Context):
              '--runstate-dir.\n\nThe default is "require_file" when the '
              '--security option is set to "strict", and "generate" when the '
              '--security option is set to "insecure_dev_mode"'),
+    click.option(
+        '--jwt-sub-allowlist-file',
```

Comment:
nit:

```suggestion
        '--jwt-sub-allow-list-file',
```

to look nice along `--jwt-revocation-list-file`

Reply:
Allowlist is a single word (instead of whitelist).  Also, this is a backport from master.